osx/KBKit/KBKit/Component/KBRedirector.m

```diff
@@ -46,7 +46,7 @@ - (void)install:(KBCompletion)completion {
   DDLogDebug(@"Starting redirector: %@", params);
   [self.helperTool.helper sendRequest:@"startRedirector" params:@[params] completion:^(NSError *err, id value) {
     if (!err) {
-      completion(nil, value);
+      completion(nil);
```

Comment:
Hmm what is this value used for and how did it not break stuff?

Reply:
That was a bug that I both introduced and fixed today.  I was coding on Linux so had to commit to test compiling on macOS.  I'll squash down those commits all together so it's not so obvious how much I suck.